packages/backend/src/models/SavedChartModel.ts

```diff
@@ -963,6 +964,115 @@ export class SavedChartModel {
         }
     }
 
+    async findChartsForValidation(projectUuid: string): Promise<
+        Array<{
+            uuid: string;
+            name: string;
+            tableName: string;
+            filters: Filters;
+            dimensions: string[];
+            metrics: string[];
+            tableCalculations: string[];
+            customMetrics: string[];
+            customMetricsBaseDimensions: string[];
+            customBinDimensions: string[];
+            customSqlDimensions: string[];
+            sorts: string[];
+        }>
+    > {
+        return this.database(SavedChartsTableName)
+            .select({
+                uuid: 'saved_queries.saved_query_uuid',
+                name: 'saved_queries.name',
+                tableName: 'saved_queries_versions.explore_name',
```

Comment:
:+1:  this is ok because we're grouping and limiting versions to 1. same with filters